pybossa/view/projects.py

```diff
@@ -3094,9 +3094,12 @@ def sync_project(short_name):
                 'A request error occurred while syncing {}: {}'
                 .format(project.short_name, str(res.__dict__)))
             msg = gettext(
```

Comment:
Considering adding 
`current_app.logger.info('response code: {}'.format(str(res.status_code))`

Reply:
Okay, I will add this. Do you think that i should add logger.info at all instances which already contain logger.error (such as the one seen here?)

Reply:
Yea that would be great